rflx/expression.py

```diff
@@ -618,19 +732,59 @@ def representation(self) -> str:
 
 
 class Size(Attribute):
-    pass
+    def variables(self, proof: bool = False) -> List['Variable']:
+        if proof:
+            if not isinstance(self.name, str):
+                raise TypeError
+            return [Variable(self.name + '__Size')]
+        return []
+
+    def z3expr(self) -> z3.ExprRef:
+        if not isinstance(self.name, str):
+            raise TypeError
+        return z3.Int(self.name + '__Size')
```

Comment:
Why not using a generic implementation in `Attribute` instead? The name of the class can be referenced by `self.__class__.__name__`.

Reply:
Good idea. Realized in 797c086.